munimap/config.py

```diff
@@ -172,8 +172,9 @@ class DefaultConfig(object):
 
     COORD_TRANSFORM_URL = ''
 
-here = path.dirname(__file__)
+    JSON_SORT_KEYS = False
 
+here = path.dirname(__file__)
```

Comment:
Minor: Indentation seems off

Reply:
Was outside the class. Moved it inside the class, since is only used there